statsmodels/examples/ex_scatter_ellipse.py

```diff
@@ -15,20 +12,14 @@
 
 nvars = 6
 mmean = np.arange(1.,nvars+1)/nvars * 1.5
-rho = 0.5
-#dcorr = rho*np.ones((nvars, nvars)) + (1-rho)*np.eye(nvars)
-r = np.random.uniform(-0.99, 0.99, size=(nvars, nvars))
-##from scipy import stats
-##r = stats.rdist.rvs(1, size=(nvars, nvars))
-r = (r + r.T) / 2.
-assert np.allclose(r, r.T)
-mcorr = r
-mcorr[lrange(nvars), lrange(nvars)] = 1
-#dcorr = np.array([[1, 0.5, 0.1],[0.5, 1, -0.2], [0.1, -0.2, 1]])
-mstd = np.arange(1.,nvars+1)/nvars
-mcov = mcorr * np.outer(mstd, mstd)
```

Comment:
This looks like an attempt to generate a random covariance matrix.  This method fails for n>2, roughly because corr(A,B) and corr(B,C) constrains corr(A,C)